tests/v1/models/test_correction.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from pydantic import ValidationError
+
+from app.v1.models.correction import Correction
+
+
+def test_valid_correction():
```

Comment:
Why not used `correction_dict` from `conftest.py`? tbh, I don't feel that having `correction_dict` and `ayah_match_dict` in the `conftest.py` is appropriate.

If you want to keep the fixtures in `conftest.py`, then let's make sure we are consistent and using this approach in all tests.